waspc/test/Generator/WriteFileDraftsTest.hs

```diff
@@ -20,10 +20,24 @@ genMockTextFileDrafts n =
           }
    in take n (map genMockTextFileDraft [1 :: Int ..])
 
+-- Returns [FileDraft] with duplicated dest paths if isDuplicate is set.
+genMockFileDrafts :: Bool -> [FileDraft]
+genMockFileDrafts isDuplicate =  case isDuplicate of
```

Comment:
Nice job getting tests going @InfiniteVerma! In this case, this helper feels pretty specific to `spec_WriteDuplicatedDstFileDrafts` I'd probably recommend removing it and just constructing the data for the test in the spec. It should just be adding 1 `let` line inside each `it` 👍🏻 Nice job!

Reply:
Makes sense! I've updated `spec_WriteDuplicatedDstFileDrafts`.